adapter/integration_tests/projects/simple_project/dbt_project.yml

```diff
@@ -10,6 +10,9 @@ snapshot-paths: ["snapshots"]
 
 target-path: "{{ env_var('temp_dir') }}/target"
 
+vars:
+  fal-scripts-paths: "fal_scripts"
+
```

Comment:
We were already setting this in line 19 of this file. Which I guess overwrites this value and that is why it works.